common/src/test/scala/uk/ac/wellcome/platform/archive/common/bagit/services/azure/AzureBagReaderTest.scala

```diff
@@ -0,0 +1,34 @@
+package uk.ac.wellcome.platform.archive.common.bagit.services.azure
+
+import uk.ac.wellcome.fixtures.TestWith
+import uk.ac.wellcome.platform.archive.common.bagit.services.{BagReader, BagReaderTestCases}
+import uk.ac.wellcome.platform.archive.common.fixtures.azure.AzureBagBuilder
+import uk.ac.wellcome.storage.azure.{AzureBlobLocation, AzureBlobLocationPrefix}
+import uk.ac.wellcome.storage.fixtures.AzureFixtures.Container
+import uk.ac.wellcome.storage.store.TypedStore
+import uk.ac.wellcome.storage.store.azure.AzureTypedStore
+
+class AzureBagReaderTest extends BagReaderTestCases[
+  Unit,
+  Container,
+  AzureBlobLocation,
+  AzureBlobLocationPrefix
+]
+  with AzureBagBuilder {
+  override def asLocation(root: AzureBlobLocationPrefix, path: String): AzureBlobLocation = root.asLocation(path)
```

Comment:
Note: this method can be pushed down into BagReaderTestCases, because this is now a required method on Prefix[Location], but let's do that separately.